src/app/components/PageContent.js

```diff
@@ -0,0 +1,116 @@
+import createVisualImage from './VisualImage';
+import createButtonRed from './ButtonRed';
+import createWhiteButtonWithIcon from './ButtonWhiteWithIcon';
+import createGameModeName from './GameModeName';
+import createModeRules from './ModeRules';
+import createScoreTable from './ScoreTable';
+import { PEOPLE, STARSHIPS, VEHICLES } from '../constants';
+import elementFactory from '../utils/elementFactory';
+
+const section = {};
+const RULES = '<span></span> Rules';
```

Comment:
Empty span is no the best solution, but we don’t have the time for it right now.